src/lib/client/sdk.gen.ts

```
@@ -0,0 +1,826 @@
+                },
+                {
+                    scheme: 'bearer',
+                    type: 'http'
+                }
+            ],
+            url: '/api/v1/scrape/scrape/complete_session/{session_id}',
+            ...options
+        });
+    }
+    
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)